app.py

```diff
@@ -184,28 +176,102 @@ def main():
     pull_cache_from_hub = arguments.pull_cache_from_hub
```

Comment:
Note that `live` and `pull_cache_from_hub` are defined here -- these need to be used later (now on line 211; I made a separate comment).

Reply:
Done!